modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Gcal.scala

```diff
@@ -0,0 +1,94 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.seqexec.server.ConfigUtilOps._
+import edu.gemini.seqexec.server.GcalController._
+import edu.gemini.spModel.config2.Config
+import edu.gemini.spModel.gemini.calunit.CalUnitConstants._
+import edu.gemini.spModel.gemini.calunit.CalUnitParams.Lamp
+import edu.gemini.spModel.seqcomp.SeqConfigNames.{CALIBRATION_CONFIG_NAME, CALIBRATION_KEY}
+
+import scala.collection.JavaConversions._
+import scala.Function.const
+import scalaz.Scalaz._
+import scalaz.\/
+
+/**
+  * Created by jluhrs on 3/21/17.
+  */
+final case class Gcal(controller: GcalController, isCP: Boolean) extends System {
+  import Gcal._
+
+  override val name: String = CALIBRATION_CONFIG_NAME
+
+  /**
+    * Called to configure a system, returns a Task[SeqexecFailure \/ ConfigResult]
+    */
+  override def configure(config: Config): SeqAction[ConfigResult] = ^(controller.getConfig, fromSequenceConfig(config, isCP))(diffConfiguration)
```

Comment:
I also would like to start using Applicatives more frequently instead of Monads where possible, but I still don't know which Applicative operators are more idiomatic in Scala. That's the first time I've seen the `^`. Do you usually see it more in the wild than the `|@|` operator? what about `<*>`?

It's a pity the syntax for Applicatives is so weird, they are very handy IMHO.